pkg/operator/api/resource/type.go

```diff
@@ -48,26 +46,16 @@ var (
 		"unknown",
 		"deployment",
 		"api",
-		"python package",
 	}
 
 	userFacingPlural = []string{
 		"unknowns",
 		"deployments",
 		"apis",
-		"python packages",
-	}
-
-	typeAcronyms = map[string]Type{
```

Comment:
I think we should still leave this as an empty map (and bring back lines 84-86) for now